src/rpcrawtransaction.cpp

```diff
@@ -435,7 +442,8 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
     UniValue inputs = params[0].get_array();
     UniValue sendTo = params[1].get_obj();
 
-    CMutableTransaction rawTx;
+    CMutableTransaction rawTx = CreateNewContextualCMutableTransaction(
+        Params().GetConsensus(), chainActive.Height() + 1);
```

Comment:
Lots of scripts out there probably already use `createrawtransaction`, on demand, before sending soon after.   I would guess this is the most popular use-case.  Requiring a default would break existing scripts.

So not needing to pass in another parameter, and defaulting to `chainActive.Height() + 1` seems a reasonable option to keep things running.  

To give users control over this, there could be an optional height field.
`createrawtransaction txs addresses [height]`

That might confuse users though as they might mix it up with expiry height.  Speaking of which... do we need a parameter for that here, or is that in the tx expiry PR?
`createrawtransaction txs addresses [expiryheight] [consensusheight]`

Since this is a low level routine, is this better?
`createrawtransaction txs addresses overwintered`

where `overwintered` is an optional json string, containing non-optional fields:
```
{
"branchid": branchid OR "consensusheight": height
"expiryheight": height
}
```
and if this overwintered json string is not present, we default to using `chainActive.Height` to determine the tx version, and for expiry height add the default value.

Reply:
Another thing to note is that upstream introduced new positional arguments:

```
createrawtransaction inputs outputs ( locktime ) ( replaceable )
```

They also introduced the ability to use named arguments.

Reply:
For 1.0.15, let's use ``chainActive.Height() + 1``. Then for 1.1.0, we can pull in the named arguments PR(s) and add a named ``consensusheight`` (or ``consensusHeight``) argument, with ``chainActive.Height() + 1`` as the default.

Reply:
Filed https://github.com/zcash/zcash/issues/2952